configs/default/PYRO-PYRODRONEF7.config

```diff
@@ -0,0 +1,115 @@
+# Betaflight / STM32F7X2 (S7X2) 4.1.0 Oct 16 2019 / 00:02:08 (714f27b2d) MSP API: 1.42
```

Comment:
The file name is wrong - should be `PYDR-PYRODRONEF7.config`.

Reply:
changed. sorry for the mistake.